### PR TITLE
fix: Update TM bulk sell for 2.1 [skip ci]

### PR DESCRIPTION
**Pull request by ryanzhoudev**
The sell buttons moving a bit is just to account for the new texture.

### CODE REVIEW

File: common/src/main/java/com/wynntils/features/trademarket/TradeMarketBulkSellFeature.java [hunk 1]
```diff
@@ -56,25 +56,28 @@ public void onSellDialogueUpdated(ContainerSetSlotEvent.Pre e) {
...
-        if (!title.equalsString(SELL_DIALOGUE_TITLE, PartStyle.StyleType.NONE)) return;
+        // The sell title changes based on the state of the screen but always contains this string
+        if (!title.contains(SELL_DIALOGUE_TITLE)) return;
 
         StyledText amountItemName = StyledText.fromComponent(
                 containerScreen.getMenu().getSlot(AMOUNT_ITEM_SLOT).getItem().getHoverName());
         if (!amountItemName.equalsString(CLICK_TO_SET_AMOUNT, PartStyle.StyleType.NONE)) return;
 
         String soldItemName = getSoldItemName(containerScreen);
+        System.out.println(soldItemName);
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/features/trademarket/TradeMarketBulkSellFeature.java]: debug

File: common/src/main/java/com/wynntils/features/trademarket/TradeMarketBulkSellFeature.java [hunk 2]
```diff
@@ -56,25 +56,28 @@ public void onSellDialogueUpdated(ContainerSetSlotEvent.Pre e) {
...
 
         removeSellButtons(containerScreen);
 
         if (soldItemName == null) return;
 
         addSellButtons(containerScreen, soldItemName);
+        System.out.println("Added sell buttons");
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/features/trademarket/TradeMarketBulkSellFeature.java]: debug

File: common/src/main/java/com/wynntils/features/trademarket/TradeMarketBulkSellFeature.java [hunk 3]
```diff
@@ -56,25 +56,28 @@ public void onSellDialogueUpdated(ContainerSetSlotEvent.Pre e) {
...
     }
 
     @SubscribeEvent
     public void onChatMessage(ChatMessageReceivedEvent e) {
         if (!sendAmountMessage) return;
-        if (!e.getOriginalStyledText().equalsString(TYPE_SELL_AMOUNT, PartStyle.StyleType.NONE)) return;
+        if (!e.getOriginalStyledText().contains(TYPE_SELL_AMOUNT)) return;
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/features/trademarket/TradeMarketBulkSellFeature.java:77]: StartsWIth perhaps? although I wouldn't be sure the symbols are always the long one, maybe it's a "second" message in a line, so perhaps adapt the regex?

File: common/src/main/java/com/wynntils/features/trademarket/TradeMarketBulkSellFeature.java [hunk 4]
```diff
@@ -56,25 +56,28 @@ public void onSellDialogueUpdated(ContainerSetSlotEvent.Pre e) {
         if (!(McUtils.mc().screen instanceof ContainerScreen containerScreen)) return;
 
         StyledText title = StyledText.fromComponent(containerScreen.getTitle());
-        if (!title.equalsString(SELL_DIALOGUE_TITLE, PartStyle.StyleType.NONE)) return;
+        // The sell title changes based on the state of the screen but always contains this string
+        if (!title.contains(SELL_DIALOGUE_TITLE)) return;
```
Comment (ShadowCat117) [inline common/src/main/java/com/wynntils/features/trademarket/TradeMarketBulkSellFeature.java]: Maybe register a container in ContainerModel that will match both, that way when the price match feature is fixed it won't need to duplicate this title check

File: common/src/main/java/com/wynntils/features/trademarket/TradeMarketBulkSellFeature.java (hunk 3 above)
Comment (ryanzhoudev) [reply common/src/main/java/com/wynntils/features/trademarket/TradeMarketBulkSellFeature.java:77]: For this, they send a block of 4 lines, and the part I'm using .contains on is guaranteed to be the second line
image

So what's there now is actually the shorter symbols, and it's guaranteed to be the shorter symbols. That's also why I didn't use startswith

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/trademarket/TradeMarketBulkSellFeature.java:77]: Okay, sure.

**Review by kristofbolyai — APPROVED** [review]

**Pull request merged**